js/character.js

```diff
@@ -0,0 +1,18 @@
+class Character {
+  constructor(name, className, hp, hpMax, mana, manaMax, rage, rageMax, energy, energyMax) {
+    this.name = name;
+    this.class = className;
+
+    this.hp = hp;
+    this.hpMax = hpMax;
+
+    this.mana = mana;
+    this.manaMax = manaMax;
+
+    this.rage = rage;
+    this.rageMax = rageMax;
+
+    this.energy = energy;
+    this.energyMax = energyMax;
+  }
+}

```

Comment:
I saw you tried to do a Creature class but wasn't quite right. This is how you are supposed to create a constructor class: you need to pass in the parameters like that, and then assign them.

So instead of creating creatures like you did with 
```
let fireElemental = new Creature();
fireElemental.max = 57;
fireElemental.min = 54;
```

You would actually do it like:

```
const fireElemental = new Creature(57, 54, ...);
```